src/mgr/Mgr.cc

```diff
@@ -591,7 +591,7 @@ void Mgr::handle_fs_map(MFSMap* m)
       c->set_default("addr", stringify(info.addr));
 
       std::ostringstream cmd;
-      cmd << "{\"prefix\": \"mds metadata\", \"who\": \""
+      cmd << "{\"prefix\": \"mds metadata\", \"role\": \""
```

Comment:
There are other instances of "mds metadata" calls in the mgr.

Reply:
@batrick I can find only [this one](https://github.com/ceph/ceph/blob/559c54f9c27d20966f63c58d6dc0da48feb14daa/src/mgr/Mgr.cc#L234) in mgr, sorry. The other comments are fixed, as you suggested.

Reply:
````
$ git grep 'mds meta' src/mgr
src/mgr/DaemonServer.cc:        oss << "{\"prefix\": \"mds metadata\", \"who\": \""
src/mgr/Mgr.cc:  mds_cmd.run(monc, "{\"prefix\": \"mds metadata\"}");
src/mgr/Mgr.cc:      cmd << "{\"prefix\": \"mds metadata\", \"who\": \""
````

Reply:
@batrick Got it. My fork was not updated and as a consequence I cloned an older version in my new temporary laptop. I fixed your comment now. Please verify.
 
Thanks.